.appveyor.yml

```diff
@@ -0,0 +1,17 @@
+os: Visual Studio 2017
+
+install:
+  - appveyor DownloadFile https://win.rustup.rs/ -FileName rustup-init.exe
+  - rustup-init -yv --default-toolchain stable --default-host x86_64-pc-windows-msvc
+  - set PATH=%PATH%;%USERPROFILE%\.cargo\bin
+  - set RUST_BACKTRACE=1
+  - rustc -vV
+  - cargo -vV
+
+build: false
+
+test_script:
+  - cargo test --all
+
+cache:
+  - '%USERPROFILE%\.cargo'
```

Comment:
```yaml
cache:
    - target -> Cargo.lock
    - C:\Users\appveyor\.cargo\registry -> Cargo.lock
```
i think it's better to use target file (some Cargo.toml in exonum's case?) and also cache `target` dir.